Assets/MixedRealityToolkit/_Core/Definitions/Utilities/InputManagerAxis.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities
+{
+    /// <summary>
+    /// Used to define an entire InputManagerAxis, with each variable defined by the same term the Inspector shows.
+    /// </summary>
+    public class InputManagerAxis
+    {
+        public string Name = string.Empty;
+        public string DescriptiveName = string.Empty;
+        public string DescriptiveNegativeName = string.Empty;
+        public string NegativeButton = string.Empty;
```

Comment:
What is a negative vs positive button?

Reply:
Ask Unity 😅

Reply:
Again, this is just how unity defines it.